connect/runtime/src/test/java/org/apache/kafka/connect/integration/ErrorHandlingIntegrationTest.java

```diff
@@ -106,7 +106,7 @@ public void testSkipRetryAndDLQWithHeaders() throws Exception {
 
         // setup connector config
         Map<String, String> props = new HashMap<>();
-        props.put(CONNECTOR_CLASS_CONFIG, "MonitorableSink");
+        props.put(CONNECTOR_CLASS_CONFIG, MonitorableSinkConnector.class.getSimpleName());
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.